<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repoWindow</class>
 <widget class="QMainWindow" name="repoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>296</width>
      <height>236</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinationEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="trafficNumEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="seatNum">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>座位号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择出行日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="carriageNumEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="departure">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>250</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>出发地：</string>
    </property>
   </widget>
   <widget class="QLabel" name="carriageNum">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车厢号(仅火车可填)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="destination">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>300</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>目的地：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="departureEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="seatNumEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="repoBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登记</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 仿宋 Std R</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COVID-19 确诊患者登记系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="trafficNum">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车次/班次编号：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="promiseBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>我已阅读并同意《疫情防控信息登记承诺书》</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>641</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
